scipy-1.0/paper.tex

```diff
@@ -720,31 +720,7 @@ \subsection*{Package organization}
 
 \subsection*{Language choices}
 
-Python, Cython, Fortran, C and C++ are the programming languages used to
-implement scientific algorithms in the SciPy library. An analysis of our codebase
-using the \texttt{linguist} library\cite{linguistref} provides a
-detailed breakdown of SciPy's composition (Table~\ref{tab:linguist}).
-
-\begin{table}
-\centering
-\begin{tabular}{p{3cm}S[table-format = <0.1]}
-\toprule
-Language & {Percent}\\
-\midrule
-Python  & 49.5 \\
-Fortran & 25.6 \\
-C       & 19.5 \\
-Cython  & 3.0  \\
-C++     & 2.3  \\[\defaultaddspace]
-{\TeX, Matlab, Shell, and Makefile} & <0.5 \\
-\bottomrule
-\end{tabular}
-\caption{Language composition of SciPy codebase: lines of code in each
-programming language as determined by the \texttt{linguist} package.
-The last row denotes tools used in supporting roles in tests,
-building, and documentation.}
-\label{tab:linguist}
-\end{table}
+According to analysis using the \texttt{linguist} library\cite{linguistref}, SciPy is approximately 5 \% Python, 25\% Fortran, 20\% C, 3\% Cython, and 2\% C++, with a dash of \TeX, MATLAB, shell script, and Make. The distribution of secondary programming languages in SciPy is a compromise between a powerful, performance-enhancing language that interacts well with Python (i.e. Cython) and the usage of languages (and their libraries) that have proven reliable and performant over many decades.
```

Comment:
Hopefully SciPy is more than 5 % Python! More seriously, I feel like we're revisiting the presentation of this data for the nth time, so if you feel strongly that it should be removed I guess you can remove it, but probably not going to sway the decision on the manuscript.

Reply:
Oops. :)

I agreed with Matt's suggestion to remove the table because of what he found in the [submission guidelines](https://www.nature.com/srep/publish/guidelines#general-figure).  Specifically, it says "Unnecessary figures should be avoided: data presented in small tables or histograms, for instance, can generally be stated briefly in the text instead."  The language breakdown table seems to fit that description exactly.